README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
+- [Headphones](https://thenounproject.com/search/?q=headphones&i=16961) icon Molly Bramlet, from the Noun Project
```

Comment:
Oh crap, I see what you mean now. Good spot. Not sure how I missed that. Will sort that next time I'm in front of the laptop.